compiler/testData/codegen/box/dataClasses/kt12330.kt

```diff
@@ -0,0 +1,14 @@
+// WITH_RUNTIME
```

Comment:
Does this box test check something which is not covered by current tests in `codegen/box/dataClasses`? If no, I propose to delete it